README.md

```diff
@@ -1,39 +1,30 @@
 # nginx_3scale agent
 nginx_3scale agent is a module that is responsible for providing authentication,
-authorization and metering of BigchainDB API users, by communicating with 3scale.
+authorization and metering of BigchainDB API users, by communicating with 3Scale.
```

Comment:
I can understand why you'd think it's written 3Scale, but all their marketing and documentation writes it as 3scale, e.g. see https://www.3scale.net/